keepalived/check/check_tcp.c

```diff
@@ -62,6 +62,24 @@ dump_tcp_check(void *data)
 	}
 }
 
+static int
```

Comment:
I prefer type bool, return true for same (currently 0), and false if different and change goto err to return false